lms/envs/bok_choy.env.json

```diff
@@ -79,7 +79,6 @@
         "ENABLE_INSTRUCTOR_ANALYTICS": true,
         "ENABLE_S3_GRADE_DOWNLOADS": true,
         "ENABLE_THIRD_PARTY_AUTH": true,
-        "ENABLE_COMBINED_LOGIN_REGISTRATION": true,
```

Comment:
Yeah I guess I'm just wondering - it seems like the settings that tests depend on should all, always, be explicitly laid out in these files rather than implicitly relying on the values in common.py. If I run my own fork and I change values in common.py then try to run tests, I'll get unexpected, hard to trace failures.